docs/deploy-full-rp-service-in-dev.md

```diff
@@ -247,9 +263,24 @@
         --service-endpoints Microsoft.ContainerRegistry
     ```
 
+1. Register your subscription with the resource provider (post directly to subscription cosmosdb container)
```

Comment:
`pkg/util/cluster/cluster.go` does this for us implicitly, but this method uses `az aro create`, which requires subscription registration within the Subscription cosmosdb container.  Hence adding.